configuration/src/main/java/io/airlift/configuration/ConfigurationInspector.java

```diff
@@ -182,8 +182,18 @@ private ConfigAttribute(String attributeName, String propertyName, String defaul
 
             this.attributeName = attributeName;
             this.propertyName = propertyName;
-            this.defaultValue = defaultValue;
-            this.currentValue = currentValue;
+            if (securitySensitive && defaultValue != null) {
+                this.defaultValue = "[REDACTED]";
+            }
+            else {
+                this.defaultValue = defaultValue;
+            }
+            if (securitySensitive && currentValue != null) {
+                this.currentValue = "[REDACTED]";
```

Comment:
For what it's worth, Platform has a `Logging.addLogTester()` mechanism for unit tests that verify logging. Added in dfef5eea7c1d4a7b9e58e1136632fcf3a5f1f7c7